src/testing/index.ts

```diff
@@ -42,6 +42,7 @@ export const stocks = {
   stocks: [
     {
       name: 'Dow Jones Industrial Average',
+      image: 'https://upload.wikimedia.org/wikipedia/en/thumb/f/f8/Dow_Jones_Logo.svg/1200px-Dow_Jones_Logo.svg.png',
```

Comment:
The images need some padding around them.  Check out the padding utility styles here: https://teradata.github.io/covalent/#/style-guide/utility-styles